ksql-rest-model/src/main/java/io/confluent/ksql/rest/SessionProperties.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest;
+
+import io.confluent.ksql.util.KsqlHostInfo;
+
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+public class SessionProperties {
```

Comment:
javadoc please :) what belongs in this? when is it refreshed? where should it be used?  also make sure to javadoc the parameters, whats the `ksqlHostInfo` - is that me? is that the caller?

Reply:
added